src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -480,6 +531,45 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 		return true;
 	}
 
+	private static void reloadScriptFiles(
```

Comment:
this isn't actually reloading anything, just loading
Also really really really needs docs with this many parameters